tests/juniper/juniper_base_protocol_test.py

```diff
@@ -1669,6 +1669,21 @@ def test_that_an_interface_cannot_be_deleted_while_there_is_still_an_rstp_config
 
         self.cleanup(reset_interface("ge-0/0/3"))
 
+    def test_that_force_up_can_be_configured(self):
```

Comment:
pretty sure you need a way to remove the force-up as well
Does it goes away when you delete the interface ? 
do we have a way of unsetting this specifically ? 

I would like to see a test where you remove it the same way we do on the switch.

Reply:
looks like  this according to your netman PR 

        <ieee-802.3ad>
            <lacp operation=\"delete\"></lacp>
        </ieee-802.3ad>

Reply:
yes, removing force-up was missed. I have added and its tests